hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestObjectStoreWithFSO.java

```diff
@@ -473,6 +473,13 @@ public void testListKeysAtDifferentLevels() throws Exception {
         ozoneBucket.listKeys("a/b3/e3", "a/b3/e3/e31.tx");
     expectedKeys = new LinkedList<>();
     checkKeyList(ozoneKeyIterator, expectedKeys);
+
+    // Key level, prefix=key case
+    ozoneKeyIterator =
+        ozoneBucket.listKeys("a/b1/c1/c1.tx");
+    expectedKeys = new LinkedList<>();
+    expectedKeys.add("a/b1/c1/c1.tx");
+    checkKeyList(ozoneKeyIterator, expectedKeys);
```

Comment:
maybe add one case for a list key on a dir prefix :
ozoneBucket.listKeys("a/b1/c1/") and one for a top level key in a bucket

Reply:
Thank you for the review! Directory case was covered in one of the existing cases-
```
ozoneKeyIterator =
        ozoneBucket.listKeys("a/b2/d1", null);
    expectedKeys = new LinkedList<>();
    expectedKeys.add("a/b2/d1/");
    expectedKeys.add("a/b2/d1/d11.tx");
    checkKeyList(ozoneKeyIterator, expectedKeys);
```
So instead, I have added a test case with trailing slash for dir prefix, and another one for key directly under the bucket.